### PR TITLE
Update Project Profile: Lucky Parking (Remove Sijia Pitts)

### LINKED ISSUE #5800: Update Project Profile: Lucky Parking (Remove Sijia Pitts)
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/lucky-parking.md` file
- [x] Observe the existing syntax of the front matter block [^1] in the file
- [x] Find the `leadership` variable and remove the following:
```
  - name: Sijia Pitts
    role: UX/UI Designer
    links:
      slack: 'https://hackforla.slack.com/team/U03CBK89KUY'
      github: 'https://github.com/sijiapitts'
    picture: https://avatars.githubusercontent.com/u/103976231
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Lucky Parking page [^2]
- [x] Once your pull request is merged, go to #4909 and
  - [x] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #4909 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        
      </details>

### Resources/Instructions
- This issue was initiated by #4909
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/lucky-parking

### PR DESCRIPTION
**Pull request by djbradleyii**
Fixes #5800

### What changes did you make?
  - Removed Sijia Pitts from the lucky-parking.md

### Why did you make the changes (we will use this info to test)?
  - We need to keep project information up to date so that visitors to the website can find accurate information.

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)

<details>
<summary>Visuals before changes are applied</summary>

before

</details>

<details>
<summary>Visuals after changes are applied</summary>
  
after

</details>

### CODE REVIEW

File: _projects/lucky-parking.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
 ---
-identification: '216854923'
```
Comment (KyleA99) [inline _projects/lucky-parking.md]: I dont believe we are supposed to change these to double quotes (not sure if this was done by a linter?).  The only changes for this PR should be what is detailed in the action items of issue #5800 

Please correct for each change

**Review by KyleA99 — CHANGES_REQUESTED** [review]
See inline comment regarding single/double quotes.

I'll review again once these issues are fixed

File: _projects/lucky-parking.md (hunk 1 above)
Comment (djbradleyii) [reply _projects/lucky-parking.md]: Oh no! Lint strikes again. I will make the update and re-request a review. Good catch.

**Comment by LRenDO** [comment]
Hi @mademarc! Thanks for volunteering to review this issue! When you have a minute, please add your ETA and Availability. If for some reason you find that you're unable to review the request, please remove yourself as a reviewer. Thanks!

**Comment by mademarc** [comment]
Review ETA: 11/16/2023
Availability: 7:10PM

**Review by mademarc — DISMISSED** [review]
Hey @djbradleyii good call on the changes for lines 58-63 and also on the adding of the double quotes either with or without linter.

**Comment by djbradleyii** [comment]
@KyleA99 good catch. I fixed the issue. Let me know if you notice anything else.

**Review by KyleA99 — DISMISSED** [review]
@djbradleyii changes look great.  Thank you for the great work!

**Review by t-will-gillis — CHANGES_REQUESTED** [review]
Hey @djbradleyii Great job on your first PR.
- Branches from/ to are good
- Linked back to original issue
- There is a concise description of what was done and why
- Relevant 'before' and 'after' screenshots are shown
- The code edits only change what needs to be changed

Everything looks good on your end- thanks!

You'll notice below that there is a merge conflict on this PR. If you have not seen this before, it just means that other people were working on the same file as you so the branch you started with is out of date. It is relatively easy to fix: First select "Resolve conflicts". The conflicts are shown on the next screen:
<details>
<summary>Merge conflicts</summary>

Screenshot 2023-11-26 143153

</details>

Your branch is showing 'Jodie Chen' and 'Yujin Chang' in Lucky Parking still, but if you look at #4909 these two have been removed by other issues. To resolve the conflict, you will want to delete these two profiles, so everything from lines 46-66. When done, select "Mark as resolved" and this should take care of the conflicts flag. 

When you are done, select the chasing arrows next to my name to "re-request review".

Thanks for working on this issue!

**Comment by djbradleyii** [comment]
@KyleA99 Now that the edits have been made, can you please approve the PR? Thanks.

**Review by t-will-gillis — APPROVED** [review]
Hey @djbradleyii - Awesome, thanks for fixing the merge conflict!

**Pull request merged**